CHANGES.md

```diff
@@ -1,6 +1,41 @@
 # grmtools 0.11.0 (XXXX-YY-ZZ)
 
-## APIs deprecated
+## API deprecation summary
+
+* Several of the functions / structs surrounding the compile-time construction
+  of grammars have changed: more details are given below, but in most cases
+  a `build.rs` that looks as follows:
+  ```rust
+    use lrlex::LexerBuilder;
+    use lrpar::CTParserBuilder;
+
+    fn main() -> Result<(), Box<dyn std::error::Error>> {
+        let lex_rule_ids_map = CTParserBuilder::<u8>::new_with_storaget()
+            .process_file_in_src("config.y")?;
+        LexerBuilder::new()
+            .rule_ids_map(lex_rule_ids_map)
+            .process_file_in_src("config.l")?;
+        Ok(())
+    }
+  ```
+  can be changed to:
+  ```rust
+    use lrlex::CTLexerBuilder;
+    use lrpar::CTParserBuilder;
+
+    fn main() -> Result<(), Box<dyn std::error::Error>> {
+        let cp = CTParserBuilder::<u8>::new_with_storaget()
+            .grammar_in_src_dir("config.y")?
```

Comment:
do we not need a `process()` after this?

Reply:
Oh I see `process` has been replaced in favour of `build`.

Reply:
Yep, to make clear that `build` consumes the builder.